moduler-pipelines.yml

```diff
@@ -0,0 +1,94 @@
+pool:
```

Comment:
How does this pipeline get invoked?

Reply:
This is a new CI, which only triggers when master branch gets new commits.
We have a testing repo [here](https://github.com/test-repo-arcturus/azure-sdk-for-go) which already gets this pipeline enabled